documentation/src/docs/segment-store-service.md

```diff
@@ -66,117 +96,116 @@ Segment Containers are a logical grouping of Segments, and are responsible for a
-    - `TransactionMapOperation`: Maps an Id to a Transaction and to its Parent Segment.
-    - `UpdateAttributesOperation`: Updates any attributes on a Segment.
-    - `MetadataCheckpoint`: Includes an entire snapshot of the Metadata. This can be useful during recovery. This contains all metadata up to this point, which is a sufficient base for all operations after it.
+    - `StreamSegmentAppendOperation`: Represents an append to a particular Stream Segment.
+    - `CachedStreamSegmentAppendOperation`: Same as `StreamSegmentAppendOperation`, but this is for internal use (instead of having an actual data payload, it points to a location in the Cache from where the data can be retrieved).
+    - `StreamSegmentSealOperation`: When processed, it sets a flag in the in-memory metadata that no more appends can be received. When the Storage Writer processes it, it marks the Stream Segment as read-only in Tier 2 Storage.
+    - `StreamSegmentTruncateOperation`: Truncates a Stream Segment at a particular offset. This causes the Stream Segment's `StartOffset` to change.
+    - `MergeTransactionOperation`: Indicates that a Transaction is to be merged into its parent Stream Segment.
+- **Metadata Operations** are auxiliary operations that indicate a change to the Container Metadata. They can be the result of an external operation (we received an request for a Stream Segment we never knew about before, so we must assign a "unique ID" to it) or to snapshot the entire metadata (which helps with recovery and cleaning up Tier 1 Storage). The purpose of the metadata operations is to reduce the amount of time needed for failover recovery (when needed).
+    - `StreamSegmentMapOperation`: Maps an ID to a Stream Segment Name.
+    - `TransactionMapOperation`: Maps an ID to a Transaction and to its Parent Segment.
+    - `UpdateAttributesOperation`: Updates any attributes on a Stream Segment.
+    - `MetadataCheckpoint`: Includes an entire snapshot of the metadata. This can be useful during recovery. This contains all metadata up to this point, which is a sufficient base for all operations after it.
 
 ### Durable Log
 
-The _Durable Log_ is the central component that handles all Log Operations. All Operations (which are created by the Container) are added to the _Durable Log_, which processes them in the order in which they were received. It is made up of a few other components, all of which are working towards a single goal of processing all incoming operations as quickly as possible, without compromising data integrity.
+The _Durable Log_ is the central component that handles all Log Operations. All operations (which are created by the Container) are added to the _Durable Log_, which processes them in the order in which they were received. It is made up of a few other components, all of which are working towards a single goal of processing all incoming operations as quickly as possible, without compromising data integrity.
```

Comment:
You are inconsistent in your use of capital letters for `operation`. It's either `Operation` or `operation`. We need to choose one and use the same casing throughout the document.

Reply:
will be using 👍 Operation->operation, Metadata->metadata